packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -115,25 +117,45 @@ const MT5TradeScreen: FC<MT5TradeScreenProps> = ({ mt5Account }) => {
     return (
         <div className='wallets-mt5-trade-screen'>
             <div className='wallets-mt5-trade-screen__details'>
-                <div className='wallets-mt5-trade-screen__details-description'>
-                    <div className='wallets-mt5-trade-screen__details-description__icon'>
-                        {platform === mt5Platform ? marketTypeIcon : platformIcon}
-                    </div>
-                    <div className='wallets-mt5-trade-screen__details-description__details'>
-                        <div className='wallets-mt5-trade-screen__label'>
-                            <WalletText lineHeight='3xs' size={isDesktop ? 'sm' : 'md'}>
-                                {platform === mt5Platform ? marketTypeTitle : platformTitle}{' '}
-                                {!activeWalletData?.is_virtual && details?.landing_company_short?.toUpperCase()}
+                <div className='wallets-mt5-trade-screen__details-header'>
+                    <div className='wallets-mt5-trade-screen__details-description'>
+                        <div className='wallets-mt5-trade-screen__details-description__icon'>
+                            {platform === mt5Platform ? marketTypeIcon : platformIcon}
+                        </div>
+                        <div className='wallets-mt5-trade-screen__details-description__details'>
```

Comment:
```suggestion
                        <div className='wallets-mt5-trade-screen__details-description__details'>
```
Talking about BEM, `block__element__element` is not a valid nesting pattern. An element is always part of a block, not another element.

Reply:
Updated